### ISSUE TITLE
ibm-combo-box AVT issue

### ISSUE THREAD

**Issue by aaratichavan** [IBM/carbon-components-angular#1949]
Facing accessibility issue

1. The ARIA attribute 'aria-disabled' is not valid for the element <ibm-combo-box> with ARIA role 'none'
2. An element with ARIA role 'combobox' does not have the required ARIA attribute(s): 'aria-expanded'

![image](https://user-images.githubusercontent.com/69956330/147720465-ab1b76f2-e4e3-40a0-82c3-27180cbc26ad.png)

**Comment by Akshat55** [comment]
1. aria-disabled attribute does not exist in combobox
2. combobox does now have the aria-expanded attribute
